exasol_script_languages_developer_sandbox/lib/aws_access/aws_access.py

```diff
@@ -14,13 +13,15 @@
 from exasol_script_languages_developer_sandbox.lib.aws_access.s3_object import S3Object
 from exasol_script_languages_developer_sandbox.lib.aws_access.snapshot import Snapshot
 from exasol_script_languages_developer_sandbox.lib.aws_access.stack_resource import StackResource
+from exasol_script_languages_developer_sandbox.lib.logging import get_status_logger, LogType
 from exasol_script_languages_developer_sandbox.lib.tags import create_default_asset_tag
 from exasol_script_languages_developer_sandbox.lib.export_vm.vm_disk_image_format import VmDiskImageFormat
 
 
 class AwsAccess(object):
     def __init__(self, aws_profile: Optional[str]):
         self._aws_profile = aws_profile
+        self._logger = get_status_logger(LogType.AWS_ACCESS)
```

Comment:
Does each object need its own logger

Reply:
thinking well, it's better to move the logger variable to the module as it's done in the other files.

Reply:
done